src/GraphQL/Reflection/ReflectionExtensions.cs

```diff
@@ -0,0 +1,199 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+
+namespace GraphQL.Reflection
+{
+    /// <summary>
+    /// Provides extension methods to <see cref="ParameterInfo"/> and <see cref="MemberInfo"/> instances to read nullable type reference annotations.
```

Comment:
PropertyInfo too. What about `MethodInfo`?

Reply:
```suggestion
    /// Provides extension methods to <see cref="ParameterInfo"/> and <see cref="PropertyInfo"/> instances to read nullable type reference annotations.
```
All other methods are private

Reply:
Both arguments and the return value are parameters of a member.  So the ParameterInfo extension method covers those cases while the PropertyInfo extension method covers properties.